templates/home.html

```diff
@@ -95,12 +95,12 @@
                     <div class="table-body-cell">Адрес:</div>
                     <div class="table-body-cell">
                         {{ $ac.Name }}
-                        <button onclick="fetch('http://{{ $.HostIP }}:{{ $.Port }}/door?placeID={{ $element.Place.ID }}&accessControlID={{ $ac.ID }}'); return false;">Открыть</button>
+                        <button onclick="fetch('{{ $.HassioIngress }}/door?placeID={{ $element.Place.ID }}&accessControlID={{ $ac.ID }}'); return false;">Открыть</button>
                     </div>
                 </div>
                 <div class="resp-table-row">
                     <div class="table-body-cell">Видеопоток:</div>
-                    <div class="table-body-cell">http://{{ $.HostIP }}:{{ $.Port }}/stream?cameraID={{ (index $.Cameras.Data $index).ID }}</div>
+                    <div class="table-body-cell">%%protocol%%//%%hostname%%:%%port%%/stream?cameraID={{ (index $.Cameras.Data $index).ID }}</div>
```

Comment:
I don't like `replaceAll` in html, it makes code harder to read and understand because of magic replacements. Lets create new variable `Host` in Go like `HassioIngress` and read `Host` header into it

So instead of 

https://github.com/moleus/domru/blob/7d545a713fc701d259b97f87a8a0b3235f6195e9/handlers/auth.go#L170

we can use  
```go
	host := r.Header.Get("Host")
```

and replace 

https://github.com/moleus/domru/blob/7d545a713fc701d259b97f87a8a0b3235f6195e9/handlers/auth.go#L216-L217

with 

```go 
 data := HomePageData{
   Host: host,
```